### DISCUSSION TITLE
Existing Google SSO with Slack Integration

### DISCUSSION THREAD

**Discussion by wasimb-atica** [outline/outline#7822] [Self Hosting]
Is it possible to use Slack Integration when using Google SSO for Signing-in into Outline? 
When clicking the Connect buttion while logged in as Google SSO from Slack Integration tab getting error -> `invalid client_id parameter`

**Comment by tommoor** [comment]
Yes, it's possible but you still need to provide the environment variables to enable the integration.

**Reply by wasimb-atica** [reply]
Which environment variables has to be provided to enable the slack integration?
